### ISSUE TITLE
Macro addStyleSheet : "center" not supported

### ISSUE THREAD

**Issue by loudoweb** [ianharrigan/haxeui#218]
Hi,

I try to load a style with Macros.addStyleSheet("assets/css/style.css"); but I have compile errors. It seems that one of these lines are not supported : 
textAlign: "center";
iconPosition: "center";

My style is coming from the gradient.css style (without any changes) in the following haxelib folder : haxeui\1,7,0\assets\styles\gradient

So is this a Macro error or is this a gradient.css error ?

**Comment by ianharrigan** [comment]
what about without the quotes? Ie:

```css
textAlign: center
```

If not, what is the actual error?

Cheers,
Ian

**Comment by loudoweb** [comment]
Thanks for your answer.
It seems that "center" or center give the same error : 
characters 28088-28094 : Unknown identifier : center
characters 2-49 : Called from macro here

**Comment by ianharrigan** [comment]
Hmmm, ok, ill check it out... Sorry for the late response!

**Comment by ianharrigan** [comment]
Does this still happen on the latest haxelib version (1.7.5)?

**Comment by ianharrigan** [comment]
What app are using? haxeui-test-app doesnt really get updated anymore, so i wouldnt use that (if you are)

**Comment by ianharrigan** [comment]
This should be fixed now.

**Comment by MintPaw** [comment]
Is this fixed? I get errors when trying to use center, I'm using the git master branch.
https://i.imgur.com/SguAgxK.png

**Comment by ianharrigan** [comment]
I think it should be fixed. Ill try an example later today... Do you have the XML you are also using? (will make it easier / quicker for me to get an example running)

Cheers,
Ian

**Comment by MintPaw** [comment]
Yes, in fact you can [clone this project](https://github.com/MintPaw/haxeuiTest) and build with "openfl build linux -debug" and see the error.
